conda/environments/cudf_dev_cuda11.5.yml

```
@@ -43,7 +43,8 @@ dependencies:
   - flake8=3.8.3
   - black=22.3.0
   - isort=5.10.1
-  - mypy=0.782
+  - mypy=0.971
+  - types-cachetools==5.2.1
```

I'd leave this unpinned. It's going to go out of date too quickly, and we left `cachetools` unpinned in the conda environment below.
```suggestion
  - types-cachetools
```